PocketKit/Sources/PocketKit/MyList/SearchItemsList/SearchViewModel.swift

```diff
@@ -70,12 +70,17 @@ class SearchViewModel: ObservableObject {
     func updateSearchResults(with searchTerm: String) {
         let term = searchTerm.trimmingCharacters(in: .whitespaces).lowercased()
         let shouldShowUpsell = !isPremium && selectedScope == .all
+
         guard !shouldShowUpsell else { return }
-        submitSearch(with: term)
-        showRecentSearches = false
 
-        _ = searchItems(with: searchTerm)
-        // TODO: pass results to update table
+        // TODO: Handle Offline for Premium https://getpocket.atlassian.net/browse/IN-971
```

Comment:
is this not returning properly?  If it is, than we can remove the TODO, otherwise can you make a follow up ticket to get it to work and assign it to me please?

Reply:
It doesn't account for doing local search if it is a Premium user and saves, so I added the TODO because it's part of another ticket. For online, this is returning properly

Reply:
discussed in slack to leave to TODO for now